apps/condo/domains/ticket/utils/clientSchema/Renders.tsx

```diff
@@ -11,4 +11,14 @@ export const getTicketDetailsRender = (search: FilterValue) => {
 
         return getTableCellRenderer(search, false, null, null, null, details)(trimmedDetails)
     }
+}
+
+export const getTicketClientNameRender = (search: FilterValue) => {
+    return function render (ClientName: string, ticket: ITicketUIState) {
+        const address = get(ticket, ['property', 'address'])
+        const maxClientNameLength = address ? address.length : ClientName.length
```

Comment:
all this calculations are not very clear. please add comments describing what and why is happening here.